src/test/java/org/beanpod/switchboard/dao/StreamDaoImplTest.java

```diff
@@ -88,6 +91,121 @@ public void testCreateChannel() {
     verify(streamRepository).save(streamEntity);
   }
 
+  @Test
+  public void testCreateChannelAlreadyExists() {
+    // given
+    CreateStreamRequest createStreamRequest = StreamFixture.getCreateStreamRequest();
```

Comment:
Test should be named `createStreamAlreadyExists` i think, and the rest of the test names need adjusting too.

Reply:
Done.